homeassistant/components/tts/__init__.py

```diff
@@ -413,6 +418,25 @@ def async_read_tts(self, filename):
         return (content, self.mem_cache[key][MEM_CACHE_VOICE])
 
 
+@asyncio.coroutine
+def write_tags(filename, data, provider, message, language, options):
+    """Write ID3 tags to file."""
+    import mutagen
+    import io
+
+    data_bytes = io.BytesIO()
+
+    data_bytes.write(data)
+    data_bytes.seek(0)
+
+    tts = mutagen.File(data_bytes, easy=True)
+    tts.tags['artist'] = options.get('voice', language)
+    tts.tags['album'] = provider.provider_name
```

Comment:
Maybe add default provider name, so if a provider doesn't set the attribute you won't get AttributeError?